### DISCUSSION TITLE
PS4 Controller does not pair or work

### DISCUSSION THREAD

**Discussion by SilverDesert** [darthcloud/BlueRetro#905] [Q&A]
Hi, 

The PS4 controller model CUH-ZCT2U does not pair or work with the BlueRetro. The BlueRetro's led goes blue as if it has paired with the controller but the controller's white led keeps on flashing non stop. Nothing happens if I move the controls on the controller. My BlueRetro is on the latest firmware v1.9.1_hw1. However the older PS4 controller model CUH-ZCT2E works and also the models older than that work. What could be the issue? 

I am using this BlueRetro ... https://www.aliexpress.us/item/3256804612769973.html?pdp_npi=4%40dis%21USD%21US%20%2417.91%21US%20%244.85%21%21%2117.91%214.85%21%402103200617072174051446292ee4b8%2112000030880521874%21sh%21US%210%21&spm=a2g0o.store_pc_allItems_or_groupList.new_all_items_2007603045590.1005004799084725&gatewayAdapt=glo2usa

**Comment by CorrePlayer** [comment]
When the led indicates that it's connected, try turning off the controller and turning on again pressing the PS button (only the PS button, don't hold the Share button)
This works with my fake DS4

**Comment by darthcloud** [comment]
I got a CUH-ZCT2U and it work fine with latest maybe try to update to latest v1.9.2?

Did it work on a older version?
